### PR TITLE
Fixed quotes function and added former user quote fetching

### PR DESCRIPTION
**Pull request by fractalpenguin**
Fixes the `:quotes` function that has been broken since the last update to discord.py, and implements former user quote fetching via user ID.

### CODE REVIEW

**Comment by thebeanogamer** [comment]
image
Failing tests need to be resolved before merging.
https://dev.azure.com/cyberdiscovery/Cyber%20Discovery%20Discord%20Bot/_build/results?buildId=90&view=ms.vss-test-web.build-test-results-tab

**Comment by thebeanogamer** [comment]
Also, @jos-b doesn't have approving reviews perms.

File: cdbot/cogs/fun.py [hunk 1]
```diff
@@ -12,22 +12,23 @@
...
-from discord import Embed, File, Member, Message, NotFound
-from discord.ext.commands import Bot, Cog, Context, command, has_any_role
+from discord import Embed, File, Message, NotFound
+from discord.ext.commands import Bot, Cog, Context, UserConverter, command, has_any_role
 from discord.utils import find as discord_find
 from PIL import Image, ImageDraw, ImageFont
 
 ascii_lowercase += " !?$"
 
-REACT_TRIGGERS = {
```
Comment (thebeanogamer) [inline cdbot/cogs/fun.py]: Why are we splitting some sections over multiple lines and not this one?

Comment (fractalpenguin) [reply cdbot/cogs/fun.py]: It can be put in a single line without exceeding the 120 character line limit.

Comment (thebeanogamer) [reply cdbot/cogs/fun.py]: For readability and easy expansion, this would be better split over multiple lines

Comment (thebeanogamer) [reply cdbot/cogs/fun.py]: Will approve if changed.

Comment (fractalpenguin) [reply cdbot/cogs/fun.py]: Done. The line limit should be made lower.

**Pull request merged**